mesheryctl/internal/cli/root/system/config.go

```diff
@@ -53,7 +53,10 @@ func getContexts(configFile string) ([]string, error) {
 
 	res, err := client.Do(req)
 	if err != nil {
-		return nil, err
+		log.Debug(err)
+		return nil, errors.Errorf("unable to reach Meshery Server at %s\n"+
+					  "Run `mesheryctl system status` to verify the server status or "+
+					  "`mesheryctl system check` to verify that your environment is ready.", mctlCfg.GetBaseMesheryURL())
```

Comment:
Once you have used this function just return the error like below suggested
```suggestion
		log.Debug(err)
		return nil, err
```

Reply:
Hi @suhail34,
thank you for the suggestion.

As you can see from the screenshot, the error handler method was creating a new error passing again the `err.Error()` as  'long description', and that's why it appeared in the message.

![image](https://github.com/meshery/meshery/assets/110017444/260162ae-a4d7-4de1-b3fa-f4a550714277)
H

I updated the error handler at the moment, but if you have any other suggestions, please share. 👍